pkg/webhook/server.go

```diff
@@ -184,19 +176,19 @@ func (s *Server) mutate(receivedAdmissionReview v1beta1.AdmissionReview) *v1beta
 
 	// set mesh name
 	meshName := s.Config.MeshName
-	if v, ok := pod.ObjectMeta.Annotations[meshNameAnnotation]; ok {
+	if v, ok := pod.ObjectMeta.Annotations[config.AppMeshMeshNameAnnotation]; ok {
 		meshName = v
 	}
 
 	// set egress ignored ports
-	if v, ok := pod.ObjectMeta.Annotations[egressIgnoredPortsAnnotation]; ok {
+	if v, ok := pod.ObjectMeta.Annotations[config.AppMeshEgressIgnoredPortsAnnotation]; ok {
 		egressIgnoredPorts = v
 	} else {
 		egressIgnoredPorts = "22"
```

Comment:
note for future PR, might be nice to place this string into the constants list in the config.go file